fs/utils.ts

```diff
@@ -28,13 +29,18 @@ export enum PathType {
   symlink = "symlink"
 }
```

Comment:
Probably simpler if this is just a union of strings? `type PathType = "file" | "dir" | "symlink";`